lib/Plugin/src/PluginWithConfig.hpp

```diff
@@ -333,6 +318,13 @@ class PluginWithConfig : public Plugin
     bool        m_storeConfigReq;               /**< Is requested to store the configuration in persistent memory? */
     bool        m_reloadConfigReq;              /**< Is requested to reload the configuration from persistent memory? */
 
+    /*TODO REMOVE if all plugins have it. Right now only in DateTimePlugin. */
+    bool mergeConfiguration(JsonObject& jsonMerged, const JsonObjectConst& jsonSource)

```

Comment:
What is the idea later with mergeConfiguration() method in this class? Isn't it only used between the plugin main class and its views only?